test/t_cose_sign_verify_mac_test.c

```diff
@@ -0,0 +1,337 @@
+/*
+ *  t_cose_sign_verify_mac_test.c
+ *
+ * Copyright 2019-2022, Laurence Lundblade
+ * Copyright (c) 2022, Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * See BSD-3-Clause license in README.md
+ */
+
+#include "t_cose/t_cose_mac_sign.h"
+#include "t_cose/t_cose_mac_verify.h"
+#include "t_cose/q_useful_buf.h"
+#include "t_cose_make_test_pub_key.h"
+#include "t_cose_sign_verify_mac_test.h"
+
+#include "psa/crypto.h"
+
```

Comment:
This should be the test functions for MAC for all crypto libraries, so there shouldn't be a PSA dependency here.